src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -454,7 +454,8 @@ private async Task<bool> HandleLinuxConsumptionPublish(Site functionApp, Func<Ta
             if (PublishBuildOption == BuildOption.Remote)
             {
                 await EnsureRemoteBuildIsSupported(functionApp);
-                await RemoveFunctionAppAppSetting(functionApp, Constants.WebsiteRunFromPackage, Constants.WebsiteContentAzureFileConnectionString, Constants.WebsiteContentShared);
+                // We don't need to remove WEBSITE_RUN_FROM_PACKAGE since Linux Consumption will set the app setting pointing to the latest package
```

Comment:
Looks like this comment is describing the line that was deleted above it. The current call removes `WebsiteContentAzureFileConnectionString`. Is this expected?

Reply:
Never mind. I get it now.